tests/phpunit/elementor/includes/managers/test-image.php

```diff
@@ -30,10 +35,6 @@ public function create_image() {
 		return $attachment_id;
 	}
 
-	public function delete_image( $attachment_id ) {
```

Comment:
Why? We don't need to clean up?

Reply:
WordPress cleans the database for each test.
and for the image itself, there is another method that does it.